third_party/2and3/click/decorators.pyi

```diff
@@ -83,7 +83,8 @@ def argument(
     metavar: Optional[str] = ...,
     expose_value: bool = ...,
     is_eager: bool = ...,
-    envvar: Optional[Union[str, List[str]]] = ...
+    envvar: Optional[Union[str, List[str]]] = ...,
+    autocompletion: Callable[[Any, List, str], List[Union[str, Tuple]] = ...,
```

Comment:
Can you add type arguments to List and Tuple? What is in the Lists and Tuples?

Reply:
done.